### ISSUE TITLE
opentelekomcloud_rds_instance_v3  - "enable_ssl" - flag leads to attributes not being populated on instance creation

### ISSUE THREAD

**Issue by reiSh6phoo9o** [opentelekomcloud/terraform-provider-opentelekomcloud#2201]
If `enable_ssl` - flag is set in `opentelekomcloud_rds_instance_v3` on instance creation, the resource attributes `private_ips` and `db` are not available after terraform apply. With `enable_ssl` set to false, the issue does not arise.

This breaks our terraform pipelines.

# Terraform provider version
otc provider version: `v1.35.1`
terraform version: `1.5.1`

# Affected Resource(s)
- opentelekomcloud_rds_instance_v3

# Terraform Configuration Files
```hcl
data "opentelekomcloud_vpc_v1" "default" {
  name   = "default"
  shared = true
}
 
data "opentelekomcloud_vpc_subnet_v1" "private" {
  name = "private"
}
 
resource "opentelekomcloud_networking_secgroup_v2" "main" {
  name = "otc-test-secgroup-2"
}
 
resource "opentelekomcloud_rds_instance_v3" "main" {
  name              = "otc-test-mysql-2"
  availability_zone = ["eu-de-01"]
 
  db {
    password = "12345678aA!_"
    type     = "MySQL"
    version  = "8.0"
  }
 
  volume {
    type = "COMMON"
    size = 40
  }
 
  security_group_id = opentelekomcloud_networking_secgroup_v2.main.id
  subnet_id         = data.opentelekomcloud_vpc_subnet_v1.private.id
  vpc_id            = data.opentelekomcloud_vpc_v1.default.id
  flavor            = "rds.mysql.c2.medium"
 
  ssl_enable = true
}
 
# outputs
output "first_instance_ip" {
  description = "rds instance first ip adress"
  value       = opentelekomcloud_rds_instance_v3.main.private_ips[0]
}
 
output "db_user_name" {
  description = "rds superuser"
  value       = opentelekomcloud_rds_instance_v3.main.db[0].user_name
}
 
output "selected_attributes" {
  description = "show selected attributes"
  value       = <<EOT
  # opentelekomcloud_rds_instance_v3.main.db[0].user_name
  ${opentelekomcloud_rds_instance_v3.main.db[0].user_name}
 
  # opentelekomcloud_rds_instance_v3.main.private_ips[0]
  ${opentelekomcloud_rds_instance_v3.main.private_ips[0]}
  EOT
}
```

# Steps to Reproduce
1. `terraform apply`

# Expected Behavior
After terraform apply, all resource attributes should be available. Attributes should be usable in terraform outputs.

# Actual Behavior
On instance creation, the resource attributes:
- `private_ips`
- `db[*].user_name`

are not available after terraform apply, if `enable_ssl` - flag is set to `true`.
This leads to `Attempt to index null value` - errors:

```
opentelekomcloud_rds_instance_v3.main: Creating...
opentelekomcloud_rds_instance_v3.main: Still creating... [10s elapsed]
(…)
opentelekomcloud_rds_instance_v3.main: Still creating... [5m50s elapsed]
opentelekomcloud_rds_instance_v3.main: Creation complete after 6m0s [id=a1cb343548244f70848074027794898bin01]
╷
│ Error: Attempt to index null value
│
│   on main.tf line 45, in output "first_instance_ip":
│   45:   value       = opentelekomcloud_rds_instance_v3.main.private_ips[0]
│     ├────────────────
│     │ opentelekomcloud_rds_instance_v3.main.private_ips is null
│
│ This value is null, so it does not have any indices.
╵
╷
│ Error: Attempt to index null value
│
│   on main.tf line 60, in output "selected_attributes":
│   60:   ${opentelekomcloud_rds_instance_v3.main.private_ips[0]}
│     ├────────────────
│     │ opentelekomcloud_rds_instance_v3.main.private_ips is null
│
│ This value is null, so it does not have any indices.
╵
*** [apply] Error 1
```

**Comment by artem-lifshits** [comment]
Hello @reiSh6phoo9o I was able to reproduce the issue.
RDS API was updated recently and we've got a couple of issues after that. This one is related to the fact that now instance is restarted automatically after applying SSL rules.
First apply will always fail because of this issue but in 10-15 sec second apply should work fine.

Fix is on the way and will be available on next release on 29th of June.
